_posts/2016-11-17-JAX-London-2016.md

```diff
@@ -0,0 +1,691 @@
+		- No DB design required
+
+Influence on the company:
+
+- We were the disruptor, on a tight deadline
+	- Success through simplicity
+	- Something that works today is better than something that is right tomorrow
+	- Led by our customers, driven by the features that they need
+- We are undergoing inevitable complexity transitions
+	- New HR systems, management processes, multiple dev locations
+	- Standard Compliance (security, development process)
+
+
+## The Serverless Cloud - Bart Blommaerts 
+<img class="float-image p-image" src="{{ '/img/jax-london-2016/bart-blommaerts.jpg' | prepend: site.baseurl }}" alt="Bart Blommaerts" style="width: 100px"/>
+
+[Bart Blommaerts](https://twitter.com/DaggieBe) starts off explaining that serverless, although as suggested by the name, does not actually mean that you're not using any server.
+What it does mean is that the existence of a server is hidden for you.
+The name might be a bit misplaced and it would be more correct to simply call it Functions as a service (FaaS).
+Serverless or FaaS might actually be the next step in the evolution.
+
+Going serverless will lead to less worries seeing as you no longer have the server management to look out for.
+Security updates, scalability and availability is all taken care off for you by the provider.
+On the other hand, more trust must be put into the provider.
+
+The need for an ops team will not fade since you still need to take care of things such as monitoring, debugging support, memory management, application configuration and more.
+All these things could be handled by a specialised, outsourced team.
+Seeing as serverless is rather new, there is also the opportunity for tooling to be built!
+
+We can distinguish the following characteristics:
+
+- Event-driven
+- Pay per execution instead of server uptime
+- The actually invocation cost depends on how long it took to execute the call
+
+An example of the pricing of AWS:
+
+- Pay per 100ms minimum each call
+- 3.2 million free tier seconds per month
+- Outside of the free tier the price per 100ms is $0.000000208
+
+Serverless comes with seamless scaling:
+
+- No risk of under- or over-provisioning
+- Short-lived "compute containers" that: 
+    - are isolated from other functions
+	- have their resources provided from the function config
+	- may be reused
+
+Serverless is stateless, there is no state persisted in between invocations.
+In order to preserve state you can still use a db, file system or cross-application cache.
+
+Bart then listed a couple of the current providers:
+
+- AWS Lambda
+    - Integrates well with all other AWS services
+- IBM Bluemix OpenWhisk
+	- OpenWhisk is open source and available on [Github](https://github.com/openwhisk/openwhisk)
+	- The most mature solution available
+	- Better UI than AWS and also has a dashboard for monitoring
+- MS Azure Cloud Functions
+	- Mature system
+	- Different paymodel compared to AWS that could be more expensive in the long run
```

Comment:
I believe you mentioned something about MS using a different pay per use system compared to AWS?